src/js/components/modals/ServiceFormModal.js

```diff
@@ -372,14 +370,19 @@ class ServiceFormModal extends mixin(StoreMixin) {
     let {jsonDefinition, jsonMode, service} = this.state;
 
     if (jsonMode) {
+      let height = '370px';
+
+      if (this.state.errorMessage) {
+        height = '462px';
+      }
```

Comment:
Hmmm... I'm a bit worried about these hardcoded numbers. Can't we think of a more robust solution? The problem begins with having to set the Ace editor height explicitly, but doing this doesn't make things much better.

Idea: calculate the actual height of the error block and subtract that from the desired Ace editor height? 
Also, what if the error block is taller than `92px` (which seems to be the hardcoded value set in `multiple-form.less` below)? 
If I'm not wrong the error block grows in height to accomodate the amount of errors shown...